packages/flutter/lib/src/cupertino/text_selection.dart

```diff
@@ -164,15 +164,15 @@ class _ToolbarRenderBox extends RenderShiftedBox {
     child.layout(heightConstraint.enforce(enforcedConstraint), parentUsesSize: true,);
     final _ToolbarParentData childParentData = child.parentData;
 
-    final Offset localTopCenter = globalToLocal(Offset(_arrowTipX, _barTopY));
+    final Offset topCenter = Offset(_arrowTipX, _barTopY);
```

Comment:
nit: `localTopCenter` was created so `_arrowTipX` and `_barTopY` can be converted to local coordinates. Now that it's no longer needed, removing the variable all together might make the code more readable.

Reply:
Good call.